third_party/chromiumos-ec/include/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2010 The Chromium OS Authors. All rights reserved.
```

Comment:
Please move this file one level higher in the directory structure (i.e. move it into chromium-ec/)

Reply:
Is it OK that sha256.c (in the top level directory) has a different license?

Reply:
Done.

Reply:
No, I guess we should leave it as-is.

Reply:
Reverted.